src/woo_publications/publications/models.py

```diff
@@ -112,6 +120,15 @@ def get_owner(self) -> ActingUser | None:
         assert isinstance(log, TimelineLogProxy)
         return log.acting_user[0]
 
+    def clean(self):
+        super().clean()
+        if not self.pk and self.publicatiestatus == PublicationStatusOptions.revoked:
+            raise ValidationError(
+                _("You cannot create a {} publication.").format(
+                    PublicationStatusOptions.revoked.label.lower()
+                )
+            )
```

Comment:
please respect the order of methods in models to keep things tidy: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style